types/gtag.js/index.d.ts

```diff
@@ -91,4 +98,14 @@ declare namespace Gtag {
     id?: string;
     name?: string;
   }
+
+  type FieldNames = 'client_id' | 'session_id' | 'gclid';
+
+  type ConsentArg = 'default' | 'update';
+  interface ConsentParams {
+    ad_storage?: 'granted' | 'denied';
```

Comment:
According to the API docs, possible values here are 'allowed' and 'denied' for 'ad_storage': https://developers.google.com/gtagjs/reference/api#consent

Reply:
No, it's not. It says `'granted' | 'denied'` in the [docs](https://developers.google.com/gtagjs/reference/api#consent).